docs/concepts/settings.md

```diff
@@ -17,6 +17,73 @@ Settings have keys that match environment variables that can be used to override
 
 Prefect provides the ability to organize settings as [profiles](#configuration-profiles) and apply the settings persisted in a profile. When you change profiles, all of the settings configured in the profile are applied. You can apply profiles to individual commands or set a profile for your environment.
 
+## Commonly configured settings
+
+This section describes some commonly configured settings for Prefect installations. See [Configuring settings](#configuring-settings) for details on setting and unsetting configuration values.
+
+### PREFECT_API_URL
+
+The `PREFECT_API_URL` value specifies the API endpoint of your Prefect Cloud workspace or Prefect Orion API server instance.
+
+For example, using a local Prefect Orion API server instance.
+```bash
+PREFECT_API_URL="http://127.0.0.1:4200/api"
+```
+
+Using Prefect Cloud:
+
+```bash
+PREFECT_API_URL="https://api.prefect.cloud/api/accounts/[ACCOUNT-ID]/workspaces/[WORKSPACE-ID]"
+```
+
+!!! tip "`PREFECT_API_URL` setting for agents"
+    When using [work queues and agents](/concepts/work-queues/) that can create flow runs for deployments in remote environments,  [`PREFECT_API_URL`](/concepts/settings/) must be set for the environment in which your agent is running. 
+
+    If you want the agent to communicate with Prefect Cloud or a Prefect Orion API server instance from a remote execution environment such as a VM or Docker container, you must configure `PREFECT_API_URL` in that environment.
+
+### PREFECT_API_KEY
+
+The `PREFECT_API_KEY` value specifies the [API key](/ui/cloud-getting-started/#create-an-api-key) value for interacting with a Prefect Cloud workspace.
+
+```bash
+PREFECT_API_KEY="[API-KEY]"
```

Comment:
```suggestion
prefect config set PREFECT_API_KEY=your_api_key
```

Quotation marks are not strictly needed

Reply:
I'm concerned that showing the `config set` command confuses with simply showing the setting and value.